hw/ip/pwrmgr/data/pwrmgr.hjson

```diff
@@ -0,0 +1,373 @@
+      name:    "pwr_rst",
+      act:     "requester",
+      package: "pwrmgr_pkg",
+    },
+
+    { struct:  "pwr_clk",
+      type:    "req_rsp",
+      name:    "pwr_clk",
+      act:     "requester",
+      package: "pwrmgr_pkg",
+    },
+
+    { struct:  "pwr_otp",
+      type:    "req_rsp",
+      name:    "pwr_otp",
+      act:     "requester",
+      package: "pwrmgr_pkg",
+    },
+
+    { struct:  "pwr_lc",
+      type:    "req_rsp",
+      name:    "pwr_lc",
+      act:     "requester",
+      package: "pwrmgr_pkg",
+    },
+
+    { struct:  "pwr_pinmux",
+      type:    "broadcast",
+      name:    "pwr_pinmux",
+      act:     "receiver",
+      package: "pwrmgr_pkg",
+    },
+
+    { struct:  "pwr_peris",
+      type:    "broadcast",
+      name:    "pwr_peris",
+      act:     "receiver",
+      package: "pwrmgr_pkg",
+    },
+
+  ],
+
+  param_list: [
+  ],
+
+  regwidth: "32",
+  registers: [
+
+    { name: "CONTROL",
+      desc: "Control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen: "CTRL_CFG_REGWEN",
+      fields: [
+        { bits: "0",
+          hwaccess: "hrw",
+          name: "LOW_POWER_HINT",
+          desc: '''
+            The low power hint to power manager.
+            The hint is an indication for how the manager should treat the next WFI.
+            '''
+          resval: "0"
+          enum: [
+            { value: "0",
+              name: "None",
+              desc: '''
+                No low power intent
+                '''
+            },
+            { value: "1",
+              name: "Low Power",
+              desc: '''
+                Next WFI should trigger low power entry
+                '''
+            },
+          ]
+        },
+
+        { bits: "4",
+          name: "CORE_CLK_EN",
+          desc: "core clock enable during low power state",
+          resval: "0"
+          enum: [
+            { value: "0",
+              name: "Disabled",
+              desc: '''
+                Core clock disabled during low power state
+                '''
+            },
+            { value: "1",
+              name: "Enabled",
+              desc: '''
+                Core clock enabled during low power state
+                '''
+            },
+          ]
+        },
+
+        { bits: "5",
+          name: "IO_CLK_EN",
+          desc: "IO clock enable during low power state",
+          resval: "0"
+          enum: [
+            { value: "0",
+              name: "Disabled",
+              desc: '''
+                IO clock disabled during low power state
+                '''
+            },
+            { value: "1",
+              name: "Enabled",
+              desc: '''
+                IO clock enabled during low power state
+                '''
+            },
+          ]
+        },
+
+        { bits: "6",
+          name: "MAIN_PDB",
+          desc: "Main power domain power down",
+          resval: "1"
+          enum: [
+            { value: "0",
```

Comment:
o, if you notice the signal is called `pdb`.  This is actually pretty common in my experience where the power down signals are called power down 'bar'.  I can clarify that this is active low.  It might make more sense given our specific convention to call this pd_n

Reply:
A, now it makes sense. I did think about the active low behaviour and the name for a second (when I started coding HDL I actually also learned to use the `b` suffix).  `pd_n` would indeed be better aligned with our style guide but I understand that my comment is a bit late.

Reply:
i've fixed it! (but it will show up in a different PR)

Reply:
Thanks for implementing this change!